provider/data/properties/fingerprints.csv

```diff
@@ -1,117 +1,117 @@
-JoiningTypeV1, <singleton>, 7072B, 7015B, dff64b1d9027a0e6
-JoiningTypeValueToLongNameV1, <singleton>, 107B, 84B, 67264eb9f128eed3
-JoiningTypeValueToShortNameV1, <singleton>, 42B, 19B, 162afda01b84cd1f
-LineBreakNameToValueV2, <singleton>, 780B, 759B, e8e1cfec25a22d9f
-LineBreakV1, <singleton>, 15284B, 15228B, ec2f73fe17e5fb1c
-LineBreakValueToLongNameV1, <singleton>, 616B, 594B, 5d261982131cd276
-LineBreakValueToShortNameV1, <singleton>, 217B, 195B, cdf91c5f64c791b1
-LogicalOrderExceptionV1, <singleton>, 74B, 44B, d980d7d0a7c8bdcc
-LowercaseV1, <singleton>, 4082B, 4053B, d8dff016f3e85b3
-MathV1, <singleton>, 866B, 837B, 59ed427c7c89295d
-NfcInertV1, <singleton>, 5546B, 5517B, d0e4374082dbba06
-NfdInertV1, <singleton>, 2564B, 2535B, deb8336c0380bebc
-NfkcInertV1, <singleton>, 6548B, 6519B, 6da7c252e9b8730c
-NfkdInertV1, <singleton>, 3530B, 3501B, c59583c38dad3c98
-NoncharacterCodePointV1, <singleton>, 140B, 110B, aa87012a5502db42
-PatternSyntaxV1, <singleton>, 200B, 171B, f4cdedc1669761dc
-PatternWhiteSpaceV1, <singleton>, 62B, 32B, ebba28c1e60b59ed
-PrependedConcatenationMarkV1, <singleton>, 74B, 44B, 1d0bae54eec76b45
-PrintV1, <singleton>, 4430B, 4401B, d9e1592920896380
-QuotationMarkV1, <singleton>, 110B, 80B, 9c199322a9d057a8
-RadicalV1, <singleton>, 50B, 20B, 9a839a36ccb728f0
-RegionalIndicatorV1, <singleton>, 38B, 8B, e43708d16def5ac
-ScriptNameToValueV2, <singleton>, 2611B, 2590B, b465e178f65d487b
-ScriptV1, <singleton>, 25752B, 25697B, 22c8c91d530f8df7
-ScriptValueToLongNameV1, <singleton>, 2055B, 2033B, d7e6f6b1e5a0721c
-ScriptValueToShortNameV1, <singleton>, 856B, 834B, 3b291a6aedd05159
-ScriptWithExtensionsPropertyV1, <singleton>, 27554B, 27476B, ed41a2aefcf7f044
-SegmentStarterV1, <singleton>, 1406B, 1377B, 430aa62e436e6153
-SentenceBreakNameToValueV2, <singleton>, 164B, 143B, 7a8e570bdbb11805
-SentenceBreakV1, <singleton>, 13992B, 13935B, a6b131153b3e7839
-SentenceBreakValueToLongNameV1, <singleton>, 128B, 105B, 5070087b62744fab
-SentenceBreakValueToShortNameV1, <singleton>, 84B, 61B, 317ae9daab3bfd00
-SentenceTerminalV1, <singleton>, 560B, 531B, 9da7ad446075388d
-SoftDottedV1, <singleton>, 236B, 207B, cc7e00d04717c4e8
-TerminalPunctuationV1, <singleton>, 728B, 699B, 8e4704aa40b14a76
-UnifiedIdeographV1, <singleton>, 134B, 104B, 327693b6e642bf65
-UppercaseV1, <singleton>, 3968B, 3939B, a3b0459e24312c27
-VariationSelectorV1, <singleton>, 56B, 26B, d70ac3c26b8feae7
-WhiteSpaceV1, <singleton>, 92B, 62B, 8f045d65fbaa6724
-WordBreakNameToValueV2, <singleton>, 332B, 311B, 1df99611463b0be4
-WordBreakV1, <singleton>, 11104B, 11047B, 724ea714b662b22c
-WordBreakValueToLongNameV1, <singleton>, 263B, 241B, 38e07cd0b9cd7476
-WordBreakValueToShortNameV1, <singleton>, 130B, 107B, e394f7d8f0841114
-XdigitV1, <singleton>, 482B, 453B, 76fd6f78de08064c
-XidContinueV1, <singleton>, 4832B, 4803B, 140f81b7dd37d214
-XidStartV1, <singleton>, 4136B, 4107B, ed861ecc89ae0dfe
+alnum/v1, <singleton>, 4844B, 4815B, 29f3f081b0194c43
```

Comment:
these could be improved but i think the ontology here just being the property name is fine, we don't need a deeper ontology

but we should perhaps call these PropsSomething

Reply:
+1 on `props/` or `properties/`